### DISCUSSION TITLE
Busco equipo Hackathon

### DISCUSSION THREAD

**Discussion by IngCarrillo12** [Launch-X-Latam/MisionFrontEnd#514] [Team Posts]
Buenas tardes mi nombre es Juan Elías Carrillo, lx08898@innovaccion.mx y ando en búsqueda de un equipo o podemos armar uno para la Hackathon, soy estudiante de ingeniería de sistema de sexto semestre y tengo conocimientos en HTML, css, js, Java, python, MySQL, mondoDB, php.

**Comment by WenAnCC** [comment]
Hola Juan,  mi nombre es Wendy Cielo lx15539@innovaccion.mx tampoco tengo equipo formado para el Hackathon. Tengo un poco más de experiencia con HTML y CSS. Si aún quieres formar equipo estoy disponible :)

**Comment by Saulflores01** [comment]
Hola
Si aún es posible, me gustaría ser parte de tu equipo
Dejo mi correo por cualquier cosa lx04135@innovacción.mx

**Comment by ClaudiaOvOc** [comment]
Yo quiero ser parte de su equipo, les dejo mi correo ***@***.***, ojalá y aún pueda

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by PacoMedGar** [comment]
Hola, yo también ando buscando equipo, 
pacomedinagar@gmail, es mi correo, tengo experiencia en backend y front , espero que si se arme :)

**Comment by julianFanjul** [comment]
Hola a todos, me podre sumar a su equipo? soy Julian, de argentina, soy diseñador multimedial con conocimientos en herramientas de diseño y HTML, CSS, REACT, NODE.JS, MONGODB, THREE.JS, entre otros.

Les dejo mi Email si alguien me puede sumar a su equipo estaré agradecido

lx06200@innovaccion.mx

**Comment by IngCarrillo12** [comment]
@ClaudiaOvOc  No aparece tu correo podriamos escribeme al correo que deje o por medio de teams

**Comment by IngCarrillo12** [comment]
@julianFanjul @PacoMedGar  Ustedes me podrian escribir por teams si gustan, todavia tengo cupos

**Comment by 4Ndrs91** [comment]
Hola puedo unirme a su equipo? soy nuevo en programación pero voy al corriente con todo lo del frontend  mi correo es lx16112@innovaccion.mx

**Comment by vanessaurbina-tuc** [comment]
somos dos buscando equipo tmb. somos front y dis ux /ui. Hay equipo disponible??

**Comment by Gallardo1225** [comment]
Holaaa disculpen. Hace ratos envié mi correo, pero no sé si alguien me tomo en cuenta 😅
Sino ando en busca de un grupo.
Apenas estoy poniéndome al día. 😥


Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
